backend/.gitignore

```diff
@@ -1,2 +1,7 @@
 node_modules
 .env
+
+# macOS
+.DS_Store
+
+.vscode/
```

Comment:
as we discussed, PR acceptance criteria:
"Your modified files are related to your tasks."
this is not related.
I created a new issue on JIRA, and assigned it to you. fix gitignore, package.json and hello world typo on a new branch please, and do not modify anything else on this branch, only add your one endpoint for returning troops data.